curiouslearning/FeedTheMonsterJS
# Fix: Cr 151- Fixed bangla font issue

# Changes
- prompt-text.ts

# How to test
- Check level 10 and 11 in bangla language

Ref: [CR-151](https://curiouslearning.atlassian.net/jira/software/projects/CR/boards/7?selectedIssue=CR-151)


[CR-151]: https://curiouslearning.atlassian.net/browse/CR-151?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
